Mobile version

gealsanchez opened the pull request:
Two parts of the portfolio mobile version were created, the first is the profile screenshoot and the second is the works screenshoot

gealsanchez committed:
Social media icons added

gealsanchez committed:
Menu icon added

gealsanchez committed:
Background images and placeholders added

gealsanchez committed:
Index.html file added

gealsanchez committed:
Style.css file added

gealsanchez committed:
Create linters.yml

.github & workflows FOLDERS WERE CREATED 
linters.yml WAS CREATED inside workflows folder

gealsanchez committed:
Update style.css

  59:1   ✖  Unexpected duplicate selector ".l-height-42", first used at line 57   no-duplicate-selectors              
  61:11  ✖  Expected single space before "{"                                      block-opening-brace-space-before    
  61:39  ✖  Expected single space before "}" of a single-line block               block-closing-brace-space-before    
  63:19  ✖  Expected "#fff5E1" to be "#fff5e1"                                    color-hex-case                      
  73:14  ✖  Expected a trailing semicolon                                         declaration-block-trailing-semicolon
 153:32  ✖  Expected a leading zero                                               number-leading-zero                 
 165:16  ✖  Expected single space before "{"                                      block-opening-brace-space-before    
 173:22  ✖  Expected single space before "{"                                      block-opening-brace-space-before    
 174:20  ✖  Expected single space after ":" with a single-line declaration        declaration-colon-space-after       
 175:1   ✖  Unexpected missing end-of-source newline                              no-missing-end-of-source-newline

gealsanchez committed:
Update style.css

 152:40  ✖  Expected a leading zero                                          number-leading-zero             
 161:16  ✖  Expected single space before "{"                                 block-opening-brace-space-before
 169:22  ✖  Expected single space before "{"                                 block-opening-brace-space-before
 169:42  ✖  Expected single space after ":" with a single-line declaration   declaration-colon-space-after

ichala commented on index.html:
```diff
@@ -0,0 +1,96 @@
+
+    <header>
+        <nav class="navigation">
+            <p class="white-1 roboto-f l-height-24 f-size-18">John Doe</p>
```
- [ ] please note that the website logo should be an anchor tag and not a `<p>`  tag, for better user experience kindly fix it

ichala commented on index.html:
```diff
@@ -0,0 +1,96 @@
+
+<body>
+
+    <header>
```
- [ ] kindly add the correct margins to the header, and the correct background color as shown in the Figma design


**your design:**

![image](https://user-images.githubusercontent.com/89282221/193342323-4eefc9ff-9161-40c3-bab5-e8c893b6a159.png)


**figma:**
![image](https://user-images.githubusercontent.com/89282221/193342251-8ffd74fc-37a8-425f-90d8-58898e20f7a7.png)

ichala commented on index.html:
```diff
@@ -0,0 +1,96 @@
+
+    <div class="wrapper">
+
+        <div class="profile">
+            <p class="orange crete-f f-weight400 l-height-46 f-size-56">Hey There. I'm Gerson
+            <p>
+            <p class="white-1 roboto-f f-weight700 l-height-24 f-size-26">I am a software developer
+            <p>
+                <a href="https://www.linkedin.com/in/gerson-sanchez-88309b57/">
+                    <p class="white-1 popins-f l-height-24 f-size-16">I can help you build a product, feature or
+                        website. Take a
+                        look of my works. If you like what you see and have a project you need coded, don’t hesitate
+                        and
+                        contact
+                        me.
+                    </p>
+                </a>
+
+            <div class="social">
+                <ul>
+                    <li><a href="https://github.com/gealsanchez"><img src="github.svg" alt=""></a></li>
+                    <li><a href="https://www.linkedin.com/in/gerson-sanchez-88309b57/"><img src="Linkedin icon.svg"
+                                alt=""></a></li>
+                    <li><a href="https://angel.co/"><img src="angel list.svg" alt=""></a></li>
+                    <li><a href="https://twitter.com/gealsanchez"><img src="twitter.svg" alt=""></a></li>
+                    <li> <a href="https://medium.com/tag/javascript"><img src="medium.svg" alt=""></a></li>
+                </ul>
+
+            </div>
+        </div>
+
+    </div> <!-- WRAPPER END -->
```
- [ ] the `<section>` tag represents a generic standalone section of the document, so instead of a normal div kindly wrap your headline section (profile) with a section tag

ichala commented on index.html:
```diff
@@ -0,0 +1,96 @@
+    <div class="wrapper">
+
+        <div class="profile">
+            <p class="orange crete-f f-weight400 l-height-46 f-size-56">Hey There. I'm Gerson
+            <p>
```
- [ ] for better SEO performance, the main title `Hey There. I'm Gerson` should be an `h1` tag instead of `<p>` tag , kindly fix it and you can check this [link ](https://www.reliablesoft.net/h1-tag/) for more infos

ichala commented on index.html:
```diff
@@ -0,0 +1,96 @@
+        <div class="profile">
+            <p class="orange crete-f f-weight400 l-height-46 f-size-56">Hey There. I'm Gerson
+            <p>
+            <p class="white-1 roboto-f f-weight700 l-height-24 f-size-26">I am a software developer
```
- [ ] the subtitle `I am a software developer` should be a `<hx>` tag (h2..h6) instead of a paragraph tag , This way will be semantically correct and the screen readers and other technologies will thank you. so kindly fix it

ichala commented on index.html:
```diff
@@ -0,0 +1,96 @@
+            <p>
+            <p class="white-1 roboto-f f-weight700 l-height-24 f-size-26">I am a software developer
+            <p>
+                <a href="https://www.linkedin.com/in/gerson-sanchez-88309b57/">
+                    <p class="white-1 popins-f l-height-24 f-size-16">I can help you build a product, feature or
+                        website. Take a
+                        look of my works. If you like what you see and have a project you need coded, don’t hesitate
+                        and
+                        contact
+                        me.
+                    </p>
+                </a>
```
- [ ] Please note that this is a normal paragraph introduction and not a link, kindly remove the `<a>` tag since it has no purpose 🙏

ichala commented on index.html:
```diff
@@ -0,0 +1,96 @@
+
+    <div class="wrapper">
+
+        <div class="profile">
```
- [ ] kindly reduce the margin (spaces) between the content of the headline and the nav bar to match Figma design 🙏 

![image](https://user-images.githubusercontent.com/89282221/193343802-24e130df-40f1-4f57-8a50-8e1c30bf065e.png)

ichala reviewed (changes requested):
Hi @gealsanchez  , 

Good job so far! 👏👏👏 
There are some issues that you still need to work on to go to the next project but you're almost there!

![the-simpsons-homer-simpson](https://user-images.githubusercontent.com/28307960/157369993-50d5ba12-099f-4f2a-9a19-5e57ee2d1c96.gif)

## To Highlight 🎉

- Your Design is Awesome ✔️ 
- Clean Code✔️

 ## Required Changes ♻️

- [ ]  🛑 🛑 🛑  please make sure that you add the correct information inside your readme file following this [Template](https://github.com/microverseinc/readme-template/blob/master/README.md) 🛑 🛑 🛑


- [ ] Check the comments below

## Optional suggestions

- N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

gealsanchez committed:
Update README.md

gealsanchez committed:
Update style.css

color navbar 3c3a39

gealsanchez committed:
Update index.html

p tag john doe --> a tag

gealsanchez committed:
Update style.css

gealsanchez committed:
Update index.html

gealsanchez committed:
Update style.css

  63:19  ✖  Expected "#fff5E1" to be "#fff5e1"                               color-hex-case                      
  73:17  ✖  Expected a trailing semicolon                                    declaration-block-trailing-semicolon
 156:40  ✖  Expected a leading zero                                          number-leading-zero                 
 165:16  ✖  Expected single space before "{"                                 block-opening-brace-space-before    
 173:43  ✖  Expected single space after ":" with a single-line declaration   declaration-colon-space-after

gealsanchez committed:
Update style.css

 63:19  ✖  Expected "#fff5E1" to be "#fff5e1"   color-hex-case

jerryowusu commented on index.html:
```diff
@@ -0,0 +1,90 @@
+        </nav>
+    </header>
+
+    <div class="wrapper">
+
+        <section class="profile">
+            <h1 class="orange crete-f f-weight400 l-height-46 f-size-56">Hey There. I'm Gerson
+            </h1>>
+            <h2 class="white-1 roboto-f f-weight700 l-height-24 f-size-26">I am a software developer
+            </h2>
+                    <p class="white-1 popins-f l-height-24 f-size-16">I can help you build a product, feature or
+                        website. Take a
+                        look of my works. If you like what you see and have a project you need coded, don’t hesitate
+                        and
+                        contact
+                        me.
+                    </p>
+
+            <div class="social">
+                <ul>
+                    <li><a href="https://github.com/gealsanchez"><img src="github.svg" alt=""></a></li>
+                    <li><a href="https://www.linkedin.com/in/gerson-sanchez-88309b57/"><img src="Linkedin icon.svg"
+                                alt=""></a></li>
+                    <li><a href="https://angel.co/"><img src="angel list.svg" alt=""></a></li>
+                    <li><a href="https://twitter.com/gealsanchez"><img src="twitter.svg" alt=""></a></li>
+                    <li> <a href="https://medium.com/tag/javascript"><img src="medium.svg" alt=""></a></li>
+                </ul>
+
+            </div>
+        </section>
+
+    </div> <!-- WRAPPER END -->
```
[OPTIONAL] No need to wrap a section in a div tag. You can consider removing it.

jerryowusu reviewed (approved):
Hi @gealsanchez 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

![good job](https://user-images.githubusercontent.com/65233860/193359378-78c83af9-12a6-4ea6-930e-d92b56a4398c.gif)


## Highlights
All linters are passing :heavy_check_mark:
Correct GitHub workflow :heavy_check_mark:
Meaningful commit messages :heavy_check_mark:

## Optional suggestions

- [ ] [OPTIONAL] Kindly consider putting all your images and icons in a single folder to make your project look neat and more proffessional. 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
